src/Features/LanguageServer/Protocol/WellKnownLspServerKinds.cs

```diff
@@ -9,12 +9,12 @@
 
 namespace Microsoft.CodeAnalysis.LanguageServer;
 
-internal enum WellKnownLspServerKinds
+public enum WellKnownLspServerKinds
 {
     /// <summary>
     /// Roslyn LSP server for razor c# requests.
     /// </summary>
-    RazorLspServer,
+    RoslynRazorLspServer,
```

Comment:
I renamed this for clarity, but looking at it now of course it's not that clear. Open to suggestions or complete revert.